README.adoc

```diff
@@ -134,6 +134,15 @@ Next set the URL of the Gradle Enterprise instance to which the build scans shou
 
 image::enable-auto-injection.png[Enable auto-injection,{thumbnail}]
 
```

Comment:
I'd add the following section:

`===== VCS repository filtering (Beta)`

Reply:
Added